proj3_chess/player.py

```diff
@@ -0,0 +1,18 @@
+from typing import Callable
+
+
+class Player:
+    def __init__(self, direction: int, symbol: Callable[[str], str]) -> None:
```

Comment:
`symbol` is not a good name for this. It is not a symbol but a transformation performed on symbols. Find a better name